aws_lambda_builders/workflows/go_dep/workflow.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
                                             **kwargs)
 
         options = kwargs["options"] if "options" in kwargs else {}
-        handler = options.get("handler", None)
+        handler = options.get("output_executable_name", None)
```

Comment:
Sorry for the nitpick. "output" doesn't convey as much. Can we call it "function_executable_name" or "lambda_executable_name" or something more specific?

Reply:
`function_executable_name` communicates this builder is only for building functions. When we start supporting layers, this may not be true anymore. I am ok with `lambda_executable_name` but I think `artifact_executable_name` is clearer (stolen from what you suggested in the SAM CLI Go Mod PR).